### PR TITLE
Styling products

### PR DESCRIPTION
**Pull request by david-deaton**

### CODE REVIEW

File: client/components/Shop/Products/Products.jsx [hunk 1]
```diff
@@ -16,26 +19,83 @@ const ProductList = ({ currentCategory }) => {
   };
   useEffect(getProducts, []);
 
+  const ExampleStyledComponent = styled.div`
```
Comment (jessicaawhy) [inline client/components/Shop/Products/Products.jsx]: Just for code cleanliness- would be nice if everything is aligned similarly.

File: server/controllers/shopController.js [hunk 2]
```diff
@@ -4,7 +4,22 @@ const pool = require('../../db/index');
 exports.exampleGet = async (req, res) => {
   // SAMPLE QUERY
   const query = `
-    SELECT * FROM products LIMIT 10
+    SELECT * FROM products WHERE categories = 'Meals'
```
Comment (jessicaawhy) [inline server/controllers/shopController.js:7]: A get request to `/products/` currently will get only meals with this update. Should we split this between `/shop/products/` getting the individual items & `/shop/meals/` getting the meals instead? Just a suggestion.

**Review by jessicaawhy — DISMISSED** [review]

File: client/components/Shop/Products/Products.jsx (hunk 1 above)
Comment (david-deaton) [reply client/components/Shop/Products/Products.jsx]: Agreed - I initially declared this styled div as ExampleStyledComponent while playing with Styled Components, but I will update.

File: server/controllers/shopController.js (hunk 2 above)
Comment (david-deaton) [reply server/controllers/shopController.js:7]: Yep - in the end we will have two separate requests; I will add this on the next commit, for now this should work okay.

**Review by jessicaawhy — APPROVED** [review]
Same comments as before- approved for now.

**Pull request merged**